README.md

```diff
@@ -95,11 +95,11 @@ These tests are written in Golang's standard testing package and are written in
 
 ### Frontend Unit Testing
 
-While frontend unit testing of components is usually never thought of by developers still at university, it is something our teams (past and present) have given thought to. In the trial stages of frontend testing, we have adopted the use of [Vue Test Utils](https://vue-test-utils.vuejs.org/) which of writing, is still in beta development. It is Vue's official unit testing utility library.
+While frontend unit testing of components is usually never thought of by developers still at university, it is something our teams (past and present) have given thought to. We have settled on using the Cypress.io framework to run E2E tests on our system. The framework was choosen because of its ease of use and low learning curve. It fits well with our current needs to provide quality and visual assurance that our frontend code appears and is used as how an end user would interact with the website. 
 
-Testing for frontend components (for this project, those files that are in the components section), will be written in the `frontend/tests/unit` directory. When testing, we use Jest as our testing framework which has integration with Vue Test Utils.
+To get it running, go to the `frontend` folder. Make sure that you have `yarn install`ed previously when setting up to make sure that you have cypress ready to run. To launch the console please use `yarn run cypress open` which will load an application window to view tests that cypress has detected. For further information please go to the [Cypress installation documentation](https://docs.cypress.io/guides/getting-started/installing-cypress.html#Opening-Cypress).
 
```

Comment:
I'd add the following: (this might not be an issue for WSL2, will need @abiramen to confirm)

Since WSL1 does not officially support GUI applications (like Cypress), the easiest way to set this up is by installing a Windows package manager (i.e. Scoop, Chocolatey) and installing `nodejs` and `yarn` through it.

Reply:
Will get Abiram to check when he's free.